libraries/botbuilder-ai/src/qnamaker-interfaces/StrictFiltersCompoundOperationType.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Enumeration of types of StrickFilters Operation Types.
+ */
+export class StrictFiltersCompoundOperationType {
```

Comment:
>StrictFiltersCompoundOperationType [](start = 13, length = 34)

As commented in the C# version, naming is horrible.

Reply:
You should rename these the same the C# one, i.e. JoinOperators and StrictFiltersJoinOperator

---
In reply to: [486653189](https://github.com/microsoft/botbuilder-js/pull/2718#discussion_r486653189) [](ancestors = 486653189)

Reply:
File has been deleted for name changes, now.